### PR TITLE
[Term Entry] Python NumPy - Math functions .unwrap()

### LINKED ISSUE #5236: [Term Entry] Python NumPy - Math functions .unwrap()
### Reason/inspiration (optional)

We would like a new term entry on the `.hypot()` function of the [`math-methods`](https://www.codecademy.com/resources/docs/numpy/math-methods) concept in NumPy, Python. The entry should go in a new file under `docs/content/numpy/concepts/math-methods/terms/unwrap/unwrap.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by toshydev**
### Description

Added The `unwrap()` term for NumPy
Location of file: content/numpy/concepts/math-methods/terms/unwrap/unwrap.md

### Issue Solved

Closes #5236

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @toshydev ,
The entry LGTM. I have suggested a few changes to you please make them asap.

Thanks,
Savi

**Comment by toshydev** [comment]
I made the requested changes. ✅️

**Comment by mamtawardhani** [comment]
@SaviDahegaonkar kindly check this on priority :)

File: content/numpy/concepts/math-methods/terms/unwrap/unwrap.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+In NumPy, the **`.unwrap()`** function is used to unwrap a phase angle array. This function adjusts the phase angles by changing any absolute difference between consecutive angles greater than the specified threshold (default: π) by their 2π complement. It is commonly used in signal processing to correct phase angle discontinuities in wrapped data.
+
+## Syntax
+```pseudo
+numpy.unwrap(input_value, discont=pi, axis=-1, period=2*numpy.pi)
```
Comment (SaviDahegaonkar) [inline content/numpy/concepts/math-methods/terms/unwrap/unwrap.md:23]:
```suggestion
numpy.unwrap(input_value, discont=numpy.pi, axis=-1, period=2*numpy.pi)
```

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @toshydev,
LGTM! The entry looks good for second round of review.


Thanks,
Savi

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @toshydev 😄 

The entry is ready to be merged 🚀

**Pull request merged**